source/comply/legal-hold.rst

```diff
@@ -49,7 +49,7 @@ Step 3: Enable legal hold in Mattermost
 
 1. In Mattermost, go to **System Console > Environment > File Storage**.
 2. Set the **File Storage System** to **Amazon S3**.
-3. Connect the Amazon S3 storage with Mattermost by specifying the S3 bucket name, prefix, AWS region, and hostname. You may optionally specify the access keys if using IAM roles.
+3. Connect the Amazon S3 storage with Mattermost by specifying the S3 bucket name, prefix, AWS region, and hostname. You may optionally specify the access keys if using IAM roles, and you can optionally specify a separate S3 Bucket than the one configured for your Mattermost server.
```

Comment:
If you specify a custom s3 bucket, you also need to specify all of the necessary settings/credentials to connect to it. It's possible the admin would simply just want to specify a separate s3 bucket though

Reply:
Thanks, @mickmister! Agreed! When we're ready for Legal Hold to move to GA, we can provide more granular instruction in the product docs.